extensions/amp-sidebar/1.0/test/test-amp-sidebar-toolbar.js

```diff
@@ -0,0 +1,243 @@
+
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ import {adopt} from '../../../../src/runtime';
+ import {createIframePromise} from '../../../../testing/iframe';
+ import {timerFor} from '../../../../src/services';
+ import {toggleExperiment} from '../../../../src/experiments';
+ import * as sinon from 'sinon';
+ import '../amp-sidebar';
```

Comment:
Toolbar requires and amp-sidebar element to be instantiated. So we need to create one for toolbar to work